hoot-core-test/src/test/cpp/hoot/core/OsmMapTest.cpp

```diff
@@ -93,7 +93,7 @@ class OsmMapTest : public CppUnit::TestFixture
 
     ElementConverter ec(map);
     const WayMap& ways = map->getWays();
-    for (WayMap::const_iterator itw = ways.begin(); itw != ways.end(); itw++)
+    for (WayMap::const_iterator itw = ways.begin(); itw != ways.end(); ++itw)
```

Comment:
CppCheck indicates that using the prefix `++` operator is much more efficient than the postfix.